app/canard/rev2/fin_calib.h

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+*                                                                              *
+* FILE:                                                                        * 
+* 		    idle.h                                                             *
+*                                                                              *
+* DESCRIPTION:                                                                 * 
+* 		Idle state in FSM -- Logic incomplete                                  *
+*                                                                              *
+*******************************************************************************/
+
+/* Define to prevent recursive inclusion -------------------------------------*/
+#ifndef __MAIN_H
+#define __MAIN_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/*------------------------------------------------------------------------------
+ Includes                                                                    
+------------------------------------------------------------------------------*/
+#include "stm32h7xx_hal.h"
+#include "sensor.h"
+#include "servo.h"
+#include "flash.h"
+#include "sdr_error.h"
+#include "fsm_canard.h"
+
+/*------------------------------------------------------------------------------
```

Comment:
I assume you copypasted main header file, which is good. But please delete unnecessary macros for abort, from line 35 to 52